system/default_ad_api/CMakeLists.txt

```diff
@@ -8,7 +8,12 @@ ament_auto_add_library(${PROJECT_NAME} SHARED
   src/interface.cpp
 )
 
+ament_auto_add_library(localization_score SHARED
+  src/localization_score.cpp
+)
+
 rclcpp_components_register_nodes(${PROJECT_NAME} "default_ad_api::InterfaceNode")
+rclcpp_components_register_nodes(localization_score "default_ad_api::LocalizationScoreNode")
```

Comment:
It's OK to put them together in one library `${PROJECT_NAME}`.

Reply:
I put them together in ${PROJECT_NAME}